dotcom-rendering/cypress/e2e/parallel-1/sign-in-gate.cy.js

```diff
@@ -1,9 +1,43 @@
 import { disableCMP } from '../../lib/disableCMP';
 import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
-
+import { config } from '../../../fixtures/config';
+import { makeWindowGuardian } from '../../../src/model/window-guardian';
```

Comment:
Using `makeWindowGuardian` from the source code to reflect the data that is in the dom

Reply:
Found an approach that doesn't need this